packages/react-i18n/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 ### Added
 
 - Added korean eastern name formatter
+- Added `form: 'auto'` option to `formatCurrency`, to automatically select `short` or `explicit` based on `currency` and `defaultCurrency` ([#1350](https://github.com/Shopify/quilt/pull/1350))
```

Comment:
This line should be above the last version.
You need to uncomment `<!-- ## [Unreleased] -->`

Reply:
oops, nice catch